test/integration/targets/lookup_items/aliases

```diff
@@ -1,3 +1,3 @@
 shippable/posix/group2
 skip/aix
-skip/python2.6  # lookups are controller only, and we no longer support Python 2.6 on the controller
+skip/python2  # lookups are controller only, and we no longer support Python 2 on the controller
```

Comment:
Why was this change made?  We support Python 2.7 on the controller.  Assuming this was accidental.

Reply:
I am guessing they want to remove Python 2 support from `ansible-runner`.

Reply:
Oh, this definitely looks incorrect. I missed this was not for the `ansible-runner` target. This change is incorrect.

Reply:
I confirmed this test runs fine on Python 2.7 with this PR. So this change is mos def not needed.

I just need more ☕.

Reply:
It is true that ansible-runner 1.4.6 supports python2, and this would work for now.

But we are deep into the development of 2.0.0, that absolutely is going to drop py2 support.

https://github.com/ansible/ansible-runner/pull/471

I wanted this PR to upgrade runner to 2.0.0, but it's not released yet so I can't reasonably do that. Anyway, my preference would be to stop running py2 checks now, because I don't see us genuinely doing any further work to support it.